src/main/scala/cromwell/engine/workflow/SingleWorkflowRunnerActor.scala

```diff
@@ -107,7 +107,10 @@ case class SingleWorkflowRunnerActor(source: WorkflowSourceFiles,
    */
   private def outputOutputs(outputs: binding.WorkflowOutputs): Unit = {
     import cromwell.binding.values.WdlValueJsonFormatter._
-    println(outputs.toJson.prettyPrint)
+    val outputValues = outputs map {
+      case (k, CallOutput(wdlValue, hash)) => (k, wdlValue)
+    }
+    println(outputValues.toJson.prettyPrint)
```

Comment:
You're right, the Scaladoc does say it outputs to standard out.  I guess this is a feature for SWRA mode.